Sources/ObservableArray.swift

```diff
@@ -322,26 +262,24 @@ extension MutableObservableArray where Item: Equatable {
   public func replace(with array: [Item], performDiff: Bool) {
     if performDiff {
       lock.lock()
-      let diff = Array.diff(self.array, array)
-
-      var deletes: [Int] = []
-      var inserts: [Int] = []
-      deletes.reserveCapacity(diff.count)
-      inserts.reserveCapacity(diff.count)
-
-      for diffStep in diff {
-        switch diffStep {
-        case .insert(_, let index):
-          inserts.append(index)
-        case .delete(_, let index):
-          deletes.append(index)
-        }
-      }
```

Comment:
Right, so you think we can still do away with the manual collation of inserts and deletes as shown here? If I send through each "change" (insert/delete/move) but wrap each in an array does that solve the collection view issues? 

I'm not sure I follow. Do you need the inserts and deletes separated still?